components/LeagueCard/LeagueCard.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+  LeagueCard,
+  LeagueCardHeader,
+  LeagueCardTitle,
+  LeagueCardDescription,
+  LeagueCardContent,
+} from './LeagueCard';
+
+describe('LeagueCard', () => {
```

Comment:
This is not correctly tested. Currently, we're just making sure elements on the screen. That's ok, but we're not testing that the props we're passing through show up where they're supposed to, the variants of the props like isElimiated, etc.

Reply:
Tests updated in latest commit.